scripts/test-infra/verify.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash -e
+
+# Run verify steps
+make verify
```

Comment:
Missing `verify` target from the makefile(?)

Reply:
``` 
 64 verify: verify-gofmt
 65 
 66 verify-gofmt:
 67         @./scripts/verify-gofmt.sh
```

Reply:
Woops, sorry 😬

Reply:
/honk